languages/gforth/build

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e
+
+CFLAGS="$CFLAGS -std=gnu99" ./configure \
+    --prefix=/usr \
+    --without-check
+
+make -j1
```

Comment:
Can we use `make -j$(nproc)` ?

Reply:
Sadly not. Parallel builds cause issues.

Reply:
Alright